documentation/src/main/asciidoc/server_guide/server_guide.adoc

```diff
@@ -1007,3 +1046,38 @@ remoteCacheManager = new RemoteCacheManager(clientBuilder.build());
 
 Additionally, you might also want to enable client certificate authentication (and therefore also allow the use of the EXTERNAL SASL mech to authenticate clients).
 
+
+
+== Multi-tenancy
+
+Multi-tenancy allows accessing multiple containers as shown below:
+
+image::images/multi-tenancy.png.png[]
+
+Currently there are two supported protocols for accessing the data - using Hot Rod client and using REST interface.
```

Comment:
The short answer is no. 

The long answer is that you can create another `<hotrod-endpoint>` or `<rest-endpoint>` and attach the same Cache Container to it.